aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -45,9 +48,15 @@ module aptos_framework::aptos_account {
     /// Basic account creation methods.
     ///////////////////////////////////////////////////////////////////////////
 
+    inline fun register_apt(account: &signer) {
+        coin::register<AptosCoin>(account);
+    }
+
     public entry fun create_account(auth_key: address) {
         let signer = account::create_account(auth_key);
-        coin::register<AptosCoin>(&signer);
```

Comment:
This should be and just want to be a bit more cautious - can we test sponsored tx where sender's account is not created to make sure not having AptosCoin CoinStore is okay?

Reply:
I can run all move unit tests with this enabled to confirm. I need to see whether PFS needs to be created - everything works after migration is complete, but coin::transfer doesn't create PFS on demand, so that's probably needed.

note, new_accounts_default_to_fa_apt_store_enabled() flag will not be enabled until ecosystem support is complete.